README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# Welcome ! 
+My name is **Mira** Joseph and I'm a second year Sociology/Literature student in the College. 
+
```

Comment:
I really like how you separated your document into interests and hobbies! It made it super readable!